contracts/BC_fusion/StakeHub.sol

```diff
@@ -0,0 +1,876 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import "./System.sol";
+
+interface IGovBNB {
+    function sync(address[] calldata validatorPools, address account) external;
+}
+
+interface IBSCValidatorSet {
+    struct Validator {
```

Comment:
I think it can be simplified, other fields are unnecessary:
```
    struct Validator {
        address consensusAddress;
        uint64 votingPower;
    }
```

Reply:
It has to be the same to  `IBSCValidatorSet. Validator`